src/components/learning-guide-chart.coffee

```diff
@@ -10,6 +10,7 @@ FLAG_BLUE      = 'flag-blue.svg'
 FLAG_GREEN     = 'flag-green.svg'
 FLAG_YELLOW    = 'flag-yellow.svg'
 FLAG_GREY      = 'flag-grey.svg'
+ICON_PENCIL    = 'icon-pencil.svg'
```

Comment:
Since this file is not used like the other icons can you name it something different? That way it does not conflict with other icon files?